Adyen/Core/Models/DisplayInformation.swift

```diff
@@ -44,18 +56,37 @@ public struct DisplayInformation: Equatable {
     /// - Parameter footnoteText: The footnote text if any.
     /// - Parameter accessibilityLabel: An optional custom `accessibilityLabel` to use.
     /// Set this if the title / subtitle might not be sufficient enough to provide a good accessibility
+    @available(*, deprecated, renamed: "init(title:subtitle:logoName:trailingInfo:footnoteText:accessibilityLabel:)")
     public init(
         title: String,
         subtitle: String?,
         logoName: String,
         disclosureText: String? = nil,
         footnoteText: String? = nil,
         accessibilityLabel: String? = nil
+    ) {
+        self.init(
+            title: title,
+            subtitle: subtitle,
+            logoName: logoName,
+            trailingInfo: disclosureText.map { .text($0) },
+            footnoteText: footnoteText,
+            accessibilityLabel: accessibilityLabel
+        )
+    }
+    
+    public init(
+        title: String,
+        subtitle: String?,
+        logoName: String,
+        trailingInfo: TrailingInfoType?,
```

Comment:
What do you think of having `nil` as the default value?

Reply:
I wanted to but then it clashes with the other init as it has the same signature if you remove the default values and the compiler cannot decide which ones to use